test/unit/math/p5.Vector.js

```diff
@@ -1882,5 +1882,48 @@ suite('p5.Vector', function() {
         expect(p5.Vector.equals(a1, a2)).to.be.true;
       });
     });
+
+    suite('p5.Vector.clampToZero()', function() {
+      let v;
+
+      test('should clamp very small positive number of vector components to zero', function() {
+        v = new p5.Vector(0.0000000000000002, 5);
+        v.clampToZero();
+        expect(v.x).to.equal(0);
+        expect(v.y).to.equal(5);
+      });
+
+      test('should clamp very small negative number of vector components to zero', function() {
+        v = new p5.Vector(-0.0000000000000002, 5);
+        v.clampToZero();
+        expect(v.x).to.equal(0);
+        expect(v.y).to.equal(5);
+      });
```

Comment:
I believe we use ```Math.abs((val || 0) - 0) <= Number.EPSILON ? 0 : val ``` in our clampToZero function, so there's no need to check for negative values. It seems like we're testing whether Math.abs() works correctly rather than testing our p5.js function.